application/deed/views.py

```diff
@@ -81,3 +82,18 @@ def create():
         except Exception as e:
             print("Database Exception - %s" % e)
             abort(status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+@deed_bp.route('/borrowers/delete/<borrower_id>', methods=['DELETE'])
+def delete_borrower(borrower_id):
+    borrower = None
+    borrowerModel = Borrower()
+    try:
+        borrower = borrowerModel.delete(borrower_id)
+    except Exception as inst:
+        print(str(type(inst)) + ":" + str(inst))
```

Comment:
theres a separate chore to refactor existing code to the new framework as the new framework is not merged yet so can't be rebased in